src/js/components/organisms/table-compare/table-compare.scss

```diff
@@ -1,5 +1,6 @@
 .table-compare {
     position: relative;
+    margin-bottom: $buildingUnit * 2.3;
```

Comment:
This results in 27.6, i think the concept of building units is fine. But this might be a bit overdoing it.